src/test/java/Homework19.java

```diff
@@ -7,12 +7,11 @@ public class Homework19 extends BaseTest {
 public void deletePlaylist () {
 
     login("kbetestack@gmail.com", "te$t$tudent");
-    WebElement getPlaylist = driver.findElement(By.cssSelector("a[href*='22390']"));
+    WebElement getPlaylist = driver.findElement(By.cssSelector("a[href*='22380']"));
     getPlaylist.click();
-    WebElement delPlaylist = driver.findElement(By.cssSelector(".del btn-delete-playlist"));
+    WebElement delPlaylist = driver.findElement(By.cssSelector(".btn-delete-playlist"));
     delPlaylist.click();
 
-
     }
 
```

Comment:
**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)